main.css

```diff
@@ -440,3 +441,109 @@ li.skill {
   width: 327px;
   background: #dfe1e6;
 }
+
+.contact-me {
+  background-color: #6070ff;
+  border: 1px solid #cfd8dc;
+  border-radius: 100px 0 0 0;
+}
+
```

Comment:
- [ ] You have done a great job with the design. However, there are some things missing. there should be a little gap between the about section and the contact form and there is an unnecessary space in the contact form please fix that.

### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/159522402-7a6b04da-e6fa-40be-9744-4d3ebf63e733.png)


### Figma's Design

![image](https://user-images.githubusercontent.com/59416136/159522281-5aff89bf-af7c-49bf-986d-838dd4027187.png)